onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
So an implication is that for now, we'd need some sort of forking/switch logic to handle the varying names.  Not sure how that fits in with the workflow of building a Docker image.  e.g., I could envision adding logic to  `bin/build-release.sh` that populates the package name based on lookup table, and then passing that as a variable into the Docker build.